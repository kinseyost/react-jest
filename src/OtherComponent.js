import React from 'react';

export default () => {
  return (
    <div> I am a very important component </div>
  );
}
