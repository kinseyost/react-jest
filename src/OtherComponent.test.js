import React from 'react';
import OtherComponent from './OtherComponent.js';
import { shallow } from 'enzyme';
import { shallowToJson } from 'enzyme-to-json'

test('OtherComponent snapshot', () => {
  const wrapper = shallow(<OtherComponent />);
  const tree = shallowToJson(wrapper);
  expect(tree).toMatchSnapshot();
})
