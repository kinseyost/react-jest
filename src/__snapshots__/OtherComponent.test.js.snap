exports[`test OtherComponent snapshot 1`] = `
<div>
   I am a very important component 
</div>
`;
